### PR TITLE
Format Fragment2

### PR DESCRIPTION
**Pull request by andrewL-avlq**

### CODE REVIEW

File: com.avaloq.tools.ddk.xtext.generator/src/com/avaloq/tools/ddk/xtext/generator/builder/BuilderIntegrationFragment2.xtend [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Avaloq Evolution AG - initial API and implementation
+ *******************************************************************************/
+
+package com.avaloq.tools.ddk.xtext.generator.builder
+
+import com.google.inject.name.Names
+import org.eclipse.xtext.generator.Binding
```
Comment (gregdyke) [inline com.avaloq.tools.ddk.xtext.generator/src/com/avaloq/tools/ddk/xtext/generator/builder/BuilderIntegrationFragment2.xtend]: Lots of unused import warnings

Comment (andrewL-avlq) [reply com.avaloq.tools.ddk.xtext.generator/src/com/avaloq/tools/ddk/xtext/generator/builder/BuilderIntegrationFragment2.xtend]: Unused imports removed.

**Comment by andrewL-avlq** [comment]
Rebased to resolve merge conflict.

**Comment by rubenporras** [comment]
Hi Andrew,

should we rebase the PR or close it, it is three years old :)

Cheers

File: com.avaloq.tools.ddk.xtext.generator/.classpath [hunk 2]
```diff
@@ -12,7 +12,10 @@
       <accessrule kind="accessible" pattern="org/eclipse/xtext/serializer/**"/>
       <accessrule kind="accessible" pattern="org/eclipse/xtext/generator/**"/>
       <accessrule kind="accessible" pattern="org/eclipse/xtext/xtext/**"/>
+<<<<<<< Upstream, based on upstream/master
       <accessrule kind="accessible" pattern="org/eclipse/xtext/util/**"/>
+=======
+>>>>>>> ed79af9 New implementation of Format using new mwe2 fragments.
```
Comment (rubenporras) [inline com.avaloq.tools.ddk.xtext.generator/.classpath:18]: This looks like a rebase mistake, the conflicts are not yet resolved

Comment (andrewL-avlq) [reply com.avaloq.tools.ddk.xtext.generator/.classpath:18]: agreed.

**Comment by rubenporras** [comment]
We can close this as we have a similar PR merged now

**Pull request closed without merging**